src/commands/ext-dev-deprecate.ts

```diff
@@ -0,0 +1,77 @@
+import * as clc from "cli-color";
+
+import { Command } from "../command";
```

Comment:
Super nit: I prefer to move all * imports to the top of this section.

Reply:
Moved to top of local sources, I think that's what you meant?